### ISSUE TITLE
Plugin showing issue that isn't reported in the CLI version

### ISSUE THREAD

**Issue by cssinate** [microsoft/vscode-eslint#1162]
I need some help. I can't understand what I'm doing wrong here.

I have a monorepo using lerna for multiple packages. My directory structure looks like this:

```
root /
  examples/
  packages/
    components/
    docs/
    scss/
```

I started down these changes because I'm using Vue 2 for this project, but I'm getting warnings about Vue 3 issues. If, in the CLI, I go to `./packages/components` and run `npx eslint`, I get no issues. But the ESLint extension shows an error in a file inside this directory.

The `scss/` directory can use whatever it's using now, that's fine. But for `components/` and `docs/`, I need the files to use whatever settings are being used when I `cd ./packages/components && npx eslint`. 

Here are my workspace settings:

```json
{
  "eslint.workingDirectories": [
    { "directory": "./packages/components", "!cwd": true },
    { "directory": "./packages/docs", "!cwd": true },
  ]
}
```

I probably missed something stupid, but I'm still getting this error from the extension that I'm not seeing in the CLI.

**Comment by dbaeumer** [comment]
@cssinate can you please provide me with a GitHub repository I can clone that demos what you are experiencing.

**Comment by cssinate** [comment]
It was not this plugin's fault. For anyone else still having issues and banging their head against the wall:

You probably have Vetur installed, right? Vetur does some linting, but it's not very good! If you're using ESLint AND Vetur, add the following to your settings:

`"vetur.validation.template": false,`

Sorry to bother you, @dbaeumer
